CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 This changelog track changes to the struqture project starting at version v1.0.0
 
+## 1.3.1
+
+* Fixed bug allowing the construction of Hermitian operator products with annihilator index higher than creator index when there are leading equal indices.
```

Comment:
Should it be "lower" instead of "higher" here?

Reply:
Yes.